cruise.umple/src/Umple_Code.ump

```diff
@@ -315,7 +331,18 @@ class UmpleModel
       System.exit(-1);
     }
    }
-
+  
+  public Map<String,String> getTimeHashMap()
+  {
+    return hashMap;
+  }
+  
+    public String getParsingTime()
+  {
+    String parsingT = String.valueOf(t11);
+    return parsingT;
+  }
+  
```

Comment:
You don't need to define this method. If you give a proper name to the variable, Umple will generate this method automatically.